doc/sidecar-example/README.md

```diff
@@ -0,0 +1,131 @@
+# How to inject side-cars?
+
+> [!WARNING] 
+> **This feature is experimental.**
+> 
+> It allows to deploy side-car containers on the frontend and backend Deployments.
+> As it is experimental, this document may not be updated on each releases.
+> It has been added starting with version 2.x.y //TODO define version.
+>
+> The following content is provided as a basis. 
+>
+> It must be adapted by yourself to your specific use case.
+
+## Goals
+
+This document demonstrates how to setup a simple (no SSL) nginx proxy as sidecar on the frontend and backend Pods.
+It uses the values defined in the section [Side-car containers (experimental)](../../README.md#side-car-containers-experimental).
+
+### Updating values file
+
+> [!NOTE]
+> The following diffs are based on [values-custom.yaml file](../../values-custom.yaml).
+
+#### Update `services` values
+
+##### Prevent creation of Ingress for default services
+
+By defining `values.(webapp|api|files).ingress` to `null`, this will prevent the creation of Ingress for those services.
+
+```diff
+### Services host configuration
+services:
+  webapp:
+-    host: neoload-web.mycompany.com
++    ingress: null
+  api:
+-    host: neoload-web-api.mycompany.com
++    ingress: null
+  files:
+-    host: neoload-web-files.mycompany.com
++    ingress: null
+```
+
+##### Adding new services for side-cars
+
+Now let's define services for our side-cars. This will allow to create Services & Ingresses.
```

Comment:
Ingress will not be created ?

Reply:
I'm not sure to get your question here.
As soon as the new service in values have a valid ingress property an Ingress will be created by the Helm Chart.